components/config/featureFlagContext.tsx

```diff
@@ -0,0 +1,31 @@
+"use client"
+
+import { createContext, FC, PropsWithChildren, useContext, useEffect, useState } from "react";
+
+export type FeatureFlags = {
+    staticStageCount: boolean
+}
+
+const featureFlagsContext = createContext<FeatureFlags>({ staticStageCount: true });
```

Comment:
Capitalize the first Letter of FeatureFlagsContext as is is standard

Reply:
Done